test/views/loadingAndUnloading/loading/LocationViewSpec.scala

```diff
@@ -65,19 +64,8 @@ class LocationViewSpec extends InputTextViewBehaviours[String] with AppWithDefau
 
   behave like pageWithSubmitButton("Save and continue")
 
-  "when during transition" - {
-    val app = transitionApplicationBuilder().build()
-    running(app) {
-      val doc = parseView(applyView(app))
-      behave like pageWithHint(
-        doc,
-        "Enter the specific location, such as the warehouse, shed or wharf, where the goods are being loaded. This can be up to 17 characters long."
-      )
-    }
-  }
-
   "when post transition" - {
```

Comment:
Title can be removed and we don't need the val app defined here

Reply:
We can move the "behave like pageWithHint" tests outside the "transition/post-transition" blocks and get rid of those totally.